App.js

```diff
@@ -1,26 +1,26 @@
-import React from 'react'
-import styled from 'styled-components/native'
+import 'react-native-gesture-handler'
+import React  from 'react'
 
-const Container = styled.View`
-  flex: 1;
-  background-color: papayawhip;
-  justify-content: center;
-  align-items: center;
-`
+import { NavigationContainer } from '@react-navigation/native';
+import { createMaterialTopTabNavigator } from '@react-navigation/material-top-tabs';
```

Comment:
Those are outer dependencies, no need to divide them from other outer dependencies